packages/api-v2/types.ts

```diff
@@ -1624,6 +1624,10 @@ type TPrivateSocketEndpoints = {
                        * Landing Company legal name
                        */
                       name?: string;
+                      /**
```

Comment:
`api-types` version needs to be updated so that we receive products as one of the types. I updated it and faced some issues, its fix is out of scope of this PR. This type should be removed after the update.